src/open_inwoner/js/components/card/ToggleHideSelection.js

```diff
@@ -0,0 +1,54 @@
+export class ToggleHideSelection {
+  static selector = '.card--toggle-hide'
+
+  constructor() {
+    this.icon = document.querySelector('.expand-icon')
+    this.button = document.getElementById('toggle-hide-elements')
+    this.button.addEventListener('click', this.toggleHide.bind(this))
+    this.button.innerHTML =
+      this.icon.outerHTML + this.button.dataset.labelReveal
+
+    // Hide all but the first three cards by default
+    const allCards = document.querySelectorAll(ToggleHideSelection.selector)
+    const cardsTail = Array.from(allCards).slice(3)
+
+    if (cardsTail.length === 0) {
+      this.button.classList.add('hide-me')
+    } else {
+      cardsTail.forEach((element) => {
+        element.classList.add('hide-me')
+      })
+    }
+  }
+
+  toggleHide(event) {
+    event.preventDefault()
+
+    // Toggle the expand/collapse state of the button
+    const isExpanded = this.button.getAttribute('aria-expanded') === 'false'
+    this.button.setAttribute('aria-expanded', isExpanded ? 'true' : 'false')
+    this.icon.textContent = isExpanded ? 'expand_more' : 'expand_less'
+    this.button.innerHTML =
+      this.icon.outerHTML +
+      (isExpanded
+        ? this.button.dataset.labelReveal
+        : this.button.dataset.labelCollapse)
+
+    if (isExpanded) {
+      console.log(`Cards are expanded`)
+      this.button.remove()
+    }
+
+    // Toggle 'hide-me' class on cards beyond the first three
+    const allCards = document.querySelectorAll(ToggleHideSelection.selector)
+    const cardsTail = Array.from(allCards).slice(3)
+
+    cardsTail.forEach((element) => {
+      element.classList.toggle('hide-me', !isExpanded)
```

Comment:
I'm not sure if it's just me, but adding the `force` flag here actually causes the expand button to do nothing, why do we need it?

Reply:
This worked for me just a few seconds ago (and seems to have worked for Jiro). I modified the code to make the toggle work (without it the spacing/padding is messed up when the button is hidden) and show the total number of questions, as in the design. Give it another try and let me know if this works now.

Reply:
Checked it, works for me now as well :+1: